mission-control-client/src/App.js

```diff
@@ -1,36 +1,53 @@
 import React, { useEffect } from "react";
-import { Switch, Route, useLocation, Redirect } from "react-router-dom";
-import PrivateRoute from "./components/PrivateRoute";
+import {
+  Switch,
+  Route,
+  useLocation,
+  Redirect,
+  useHistory
+} from "react-router-dom";
+import PrivateRoute from "./utils/PrivateRoute";
+import SPrivateRoute from "./utils/S_PrivateRoute";
```

Comment:
It's better to be explicit and call it `StudentPrivateRoute` even if it is a bit longer to write.